11/16/16 11:49:23 AM     ValidateLogIn(): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - No connection could be made because the target machine actively refused it.)
11/16/16 11:51:42 AM     TestConn(): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - No connection could be made because the target machine actively refused it.)
11/16/16 11:52:08 AM     TestConn(): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - No connection could be made because the target machine actively refused it.)
11/16/16 11:54:21 AM     TestConn(): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 0 - No connection could be made because the target machine actively refused it.)
11/16/16 11:54:28 AM     TestConn(): Cannot open database "dbcUBP_EcardMgr" requested by the login. The login failed.
Login failed for user 'sa'.
11/16/16 12:34:45 PM     PrintPage(): CardName JUAN, GSIS#123456789  . Runtime error catched Parameter is not valid.
11/16/16 12:36:57 PM     PrintPage(): CardName JUAN, GSIS#123456789  . Runtime error catched Parameter is not valid.
11/16/16 01:28:20 PM     txtLast3Digits_KeyPress(): Runtime error catched Index was outside the bounds of the array.
11/16/16 01:28:32 PM     txtLast3Digits_KeyPress(): Runtime error catched Index was outside the bounds of the array.
